include/wlr/types/wlr_virtual_pointer_v1.h

```diff
@@ -45,6 +46,7 @@ struct wlr_virtual_pointer_v1_new_pointer_event {
 	struct wlr_virtual_pointer_v1 *new_pointer;
 	/** Suggested by client; may be NULL. */
 	struct wlr_seat *suggested_seat;
+	struct wlr_output *selected_output;
```

Comment:
Maybe name this `suggested_output`, since we already have `suggested_seat`.

Also a comment indicating that it may be NULL would be nice.